### PR TITLE
add handling for single band images in maps

### PR DESCRIPTION
**Pull request by drwelby**
Addresses #638 by adding support for single band images in slippy maps, and the ability to specify the colormap used for those images.

### CODE REVIEW

File: gbdxtools/vectors.py [hunk 1]
```diff
@@ -399,16 +400,19 @@ def map(self, features=None, query=None, styles=None,
         })
         template.inject()
 
-    def _build_image_layer(self, image, image_bounds):
+    def _build_image_layer(self, image, image_bounds, cmap):
         if image is not None:
             if isinstance(image, da.Array):
-                arr = image.rgb()
+                if len(image.shape) == 2:
```
Comment (chelm) [inline gbdxtools/vectors.py]: Will this work for the case in gbdxtools where the single band image shape is `(1, 1000, 1000)`?

**Comment by chelm** [comment]
@drwelby i wonder if when we determine that we have a single band image do we want to default `cmap` to `Greys` or something? In this version the user would need to always define cmap when using a single band image, right?

File: gbdxtools/vectors.py (hunk 1 above)
Comment (drwelby) [reply gbdxtools/vectors.py]: I wasn't sure if that ever happens, single band slices seem to be (r,c), not (1,r,c)

**Comment by drwelby** [comment]
there's a default cmap already but we could define one in the function since it's ignored in RGB mode

File: gbdxtools/vectors.py (hunk 1 above)
Comment (drwelby) [reply gbdxtools/vectors.py]: Handling this now just in case

**Comment by drwelby** [comment]
Now you can make a map just with an image (no features or queries) and it will load the map to the center of the image if the map center is not specified.

**Pull request merged**